### PR TITLE
Principles

### PR DESCRIPTION
**Pull request by mih**
Tried to reduce redundancies between preamble, principles, and
conclusion

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -236,14 +236,15 @@ This allows DataLad and many other projects to facilitate access to an ever-grow
 
 ## Design principles
 
-The development of DataLad is guided by a few fundamental principles to ensure its open and domain agnostic nature, to maximize the longterm utility of its datasets and to minimze the technical debt introduced by using it:
+Besides the free software nature and open processes of the DataLad project, the development of DataLad is guided by a four principles to ensure its open and domain agnostic nature, to maximize the long-term utility of its datasets and to minimize users' technical debt:
```
Comment (bpoldrack) [inline paper.md]:
```suggestion
Besides the free software nature and open processes of the DataLad project, the development of DataLad is guided by four principles to ensure its open and domain agnostic nature, to maximize the long-term utility of its datasets and to minimize users' technical debt:
```

File: paper.md [hunk 2]
```diff
@@ -236,14 +236,15 @@ This allows DataLad and many other projects to facilitate access to an ever-grow
...
 
-- There are only two recognized entities: datasets and files
-- A dataset is a Git repository with an optional annex
-- Minimization of custom procedures and data structures: Users must not loose data or data access, if DataLad would vanish
-- Complete decentralization, with no required central server or service, but maximum interoperability to allow (re-)use of existing 3rd-party data resources and infrastructure
+- Datasets and the files they comprise are only two recognized entities
```
Comment (bpoldrack) [inline paper.md]:
```suggestion
- Datasets and the files they comprise are the only two recognized entities
```

**Review by bpoldrack — COMMENTED** [review]
LGTM

File: paper.md (hunk 2 above)
Comment (mih) [reply paper.md]: Thx

**Comment by adswa** [comment]
Cool!

**Comment by yarikoptic** [comment]
Resolved conflict by taking this branch version of the sentence (`master` only introduced the same `*emphasis*` formatting).

**Pull request merged**